mvrx/src/main/kotlin/com/airbnb/mvrx/lifecycleAwareLazy.kt

```diff
@@ -12,13 +11,15 @@ private object UninitializedValue
 /**
  * This was copied from SynchronizedLazyImpl but modified to automatically initialize in ON_CREATE.
  */
-@RestrictTo(RestrictTo.Scope.LIBRARY)
 @SuppressWarnings("Detekt.ClassNaming")
-class lifecycleAwareLazy<out T>(private val owner: LifecycleOwner, initializer: () -> T) : Lazy<T>, Serializable {
+class lifecycleAwareLazy<out T>(private val owner: LifecycleOwner, initializer: () -> T) : Lazy<T>,
```

Comment:
Why did you unrestrict this?

Reply:
it's needed in the mocking module. this should have been changed to `Scope.LIBRARY_GROUP` though, thanks for catching